stock_full_location_reservation/models/stock_move_line.py

```diff
@@ -45,16 +45,21 @@ def _get_full_location_reservable_qties(self, package_only=None):
     def _full_location_reservation(self, package_only=None):
         reservable_qties = self._get_full_location_reservable_qties(package_only)
         moves_to_assign_ids = []
-        for line in self.exists():  # Move line should have been deleted
+        for line in self.exists():
+            # Move line should have been deleted during a move merge
+            if not line.exists():
+                continue
```

Comment:
I would avoid to call exists a first time be fore the loop and then on each loop. Just do it on each loop

Reply:
@lmignon Forgot to remove the former one :sweat_smile: